cmd/email-exporter/main.go

```diff
@@ -0,0 +1,107 @@
+package notmain
+
+import (
+	"context"
+	"flag"
+	"os"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/email"
+	emailpb "github.com/letsencrypt/boulder/email/proto"
+	bgrpc "github.com/letsencrypt/boulder/grpc"
+)
+
+// Config holds the configuration for the email-exporter service.
+type Config struct {
+	EmailExporter struct {
+		cmd.ServiceConfig
+
+		// PerDayLimit is our daily limit as determined by the tier of our
+		// Salesforce account. For more information, see:
+		// https://developer.salesforce.com/docs/marketing/pardot/guide/overview.html?q=rate%20limits
+		PerDayLimit float64 `validate:"required,min=1"`
+
+		// PardotBusinessUnit is the Pardot business unit to use.
+		PardotBusinessUnit string `validate:"required"`
+
+		// ClientId is the OAuth API client ID provided by Salesforce.
+		ClientId cmd.PasswordConfig
```

Comment:
I think the client ID in OAuth is not a secret, so this doesn't need to be a PasswordConfig. Let's double check that.

Reply:
Salesforce provides this field as a randomly generated string, which gave me the impression that it should be treated with some level of confidentiality. That said, I see your point, and if there’s a preferred approach, I’m happy to adjust accordingly.